packages/components/src/custom-select-control/test/index.js

```diff
@@ -87,4 +88,127 @@ describe( 'CustomSelectControl', () => {
 			screen.getByRole( 'button', { name: 'Custom select' } )
 		).toHaveTextContent( 'Hint' );
 	} );
+
+	it( 'Initial value should be replaced when a new item is selected', async () => {
+		const user = userEvent.setup();
+		const mock = jest.fn();
+
+		render(
+			<CustomSelectControl
+				onChange={ ( { selectedItem } ) => mock( selectedItem ) }
+				options={ options }
+				__nextUnconstrainedWidth
```

Comment:
Is `__nextUnconstrainedWidth` important for these tests? If not, should we remove it to avoid adding extra noise?

Reply:
Without it, we will get console warns